pkg/providers/tinkerbell/reconciler/reconciler.go

```diff
@@ -33,16 +41,32 @@ type IPValidator interface {
 
 // Scope object for Tinkerbell reconciler.
 type Scope struct {
-	ClusterSpec      *c.Spec
-	isRollingUpgrade bool
+	ClusterSpec   *c.Spec
+	operationType string
+	controlPlane  *tinkerbell.ControlPlane
+	workers       *tinkerbell.Workers
 }
 
 // NewScope creates a new Tinkerbell Reconciler Scope.
 func NewScope(clusterSpec *c.Spec) *Scope {
 	return &Scope{
-		ClusterSpec:      clusterSpec,
-		isRollingUpgrade: false,
+		ClusterSpec:   clusterSpec,
+		operationType: createOperation,
```

Comment:
why default to create?

Reply:
You're right, I added this to help troubleshoot some tests but my setOperationType() function was defaulting to Create if it does not detect scaling or rolling. But in reality it should stay as "" to prevent create hardware validations from being run.